src/fileoverview_comment_transformer.ts

```diff
@@ -22,13 +23,20 @@ const FILEOVERVIEW_COMMENT_MARKERS: ReadonlySet<string> =
  * Given a parsed \@fileoverview comment, ensures it has all the attributes we need.
  * This function can be called to modify an existing comment or to make a new one.
  *
+ * @param options Compiler options.
+ * @param source Original TS source file. Its path is added in \@fileoverview.
  * @param tags Comment as parsed list of tags; modified in-place.
  */
-function augmentFileoverviewComments(tags: jsdoc.Tag[]) {
+function augmentFileoverviewComments(
+    options: ts.CompilerOptions, source: ts.SourceFile, tags: jsdoc.Tag[]) {
   // Ensure we start with a @fileoverview.
   if (!tags.find(t => t.tagName === 'fileoverview')) {
     tags.splice(0, 0, {tagName: 'fileoverview', text: 'added by tsickle'});
   }
+  const fileOverview = tags.find(t => t.tagName === 'fileoverview');
+  if (fileOverview && options.rootDir != null) {
```

Comment:
Put some comment around here about how this 'generated from' string is read by other code and shouldn't be changed

Reply:
Done.